src/quilt/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+requires = ["poetry_core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "portablemc-quilt"
+version = "2.0.1"
+description = "Start Minecraft using the Quilt mod loader using '<exec> start quilt:[<mc-version>[:<loader-version>]]'."
+authors = ["Théo Rozier <contact@theorozier.fr>"]
```

Comment:
You're the author! Put your name before mine

Reply:
added